Module 13 Activity: Continuous Deployment Steps

anikeshk opened the pull request:
This PR contains the deployment instructions for MongoDB on MongoDB Atlas, and the server and the client on Render.com.

Preview: [/activities/continuous-deployment](https://deploy-preview-39--magical-haupia-72594e.netlify.app/activities/continuous-deployment) 

Pending:

1. Adding the GitHub Classroom invite link in the "GitHub Repository" section.

anikeshk committed:
Add m14 activity steps for db

anikeshk committed:
Merge branch 'neu-se:main' into main

anikeshk committed:
Add m14 activity steps for server and client

anikeshk committed:
Update m14 activity with formatting

ViharReddy commented on Activities/Module14 Activity.md:
```diff
@@ -0,0 +1,137 @@
+   5. For Root Directory, type in "client".
+   6. For Build Command, type in "npm install && npm run build".
+   7. For Publish directory, type in "build".
+   8. In the Environment Variables section, add a variable called `REACT_APP_API_URL`. For the value, add the server URL.
```
change to "REACT_APP_SERVER_URL"

anikeshk replied:
I have updated the env variable name.

anikeshk committed:
Update m14 activity with correct server url env

iamabhutta committed:
Update and rename Module14 Activity.md to Module13 Activity.md

anikeshk committed:
Update m13 activity with feedback comments

anikeshk commented:
Added the following changes:
1. Minor grammatical mistakes.
2. Specified what default collections the student will see after they connect the deployed MongoDB cluster to their MongoDB Compass.
3. Added `fake_so` to end of the connection string.
4. Re-rephrased the environmental instructions to use the `.env` file.
5. Updated the server install and start steps.
6. Add a note about server inactivity.

anikeshk committed:
Update m13 activity env variable

anikeshk committed:
Update m13 activity server instructions

anikeshk committed:
Update m13 activity note formatting

iamabhutta commented:
Closed without merge. Will work on #52 instead!
